sources/packages/backend/workflow/test/2023-2024/parttime-assessment/awards-amounts/parttime-assessment-2023-2024-awards-amount-CSPT.e2e-spec.ts

```diff
@@ -125,4 +103,37 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CS
     expect(calculatedAssessment.variables.awardEligibilityCSPT).toBe(false);
     expect(calculatedAssessment.variables.federalAwardNetCSPTAmount).toBe(0);
   });
+
+  it("Should determine federalAwardNetCSPTAmount when awardEligibilityCSPT is true and programYearTotalPartTimeCSPT is null", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataCRAReportedIncome = 20001;
+    assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+    assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+      YesNoOptions.Yes;
+    assessmentConsolidatedData.partner1CRAReportedIncome = 22999;
+    assessmentConsolidatedData.programYearTotalPartTimeCSPT = null;
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+    // Assert
+    expect(calculatedAssessment.variables.awardEligibilityCSPT).toBe(true);
+    expect(
+      calculatedAssessment.variables.federalAwardCSPTAmount,
+    ).toBeGreaterThan(100);
+    expect(calculatedAssessment.variables.federalAwardNetCSPTAmount).toBe(
+      Math.min(
+        calculatedAssessment.variables.calculatedDataTotalRemainingNeed1,
+        Math.min(
+          calculatedAssessment.variables.dmnPartTimeAwardAllowableLimits
+            .limitAwardCSPTAmount,
+          calculatedAssessment.variables.federalAwardCSPTAmount,
+        ),
+      ),
+    );
+    expect(calculatedAssessment.variables.federalAwardNetCSPTAmount).toBe(2520);
```

Comment:
I see a difference in the outputs between 2 program years for the award value for same input.
2021-2022
![image](https://github.com/bcgov/SIMS/assets/54600590/226dc69a-549d-4133-95e4-262510e6237e)

2023-2024

![image](https://github.com/bcgov/SIMS/assets/54600590/44a2b054-095f-4965-a436-dd01fa518ae8)

Reply:
The dmn which defines the limitAwardCSPTAmount is different for ProgramYears 2021-2022 & 2023-2024
![image](https://github.com/bcgov/SIMS/assets/62901416/cbc860b2-7a68-40d1-8a0b-bbc89e1e1eb2)